### PR TITLE
improved ux of quiz form so submit button turns grey when clicked and…

### PR DESCRIPTION
**Pull request by PAZHames**
… focus moves to start button, which turns yellow

working on port 3000

- deleted about page test file

### CODE REVIEW

File: client/package-lock.json
Comment (jacquelyn-bradshaw) [inline client/package-lock.json:1]: Please can you remove this as not sure why it thinks it is making 16199 changes

**Comment by PAZHames** [comment]
not super clear why it seems to have pushed the package.json...

File: client/src/pages/InstructionsPage/InstructionsPage.js
Comment (jacquelyn-bradshaw) [inline client/src/pages/InstructionsPage/InstructionsPage.js:1]: Why is so much code changing? I can't see Browser Tab component in your new code

**Review by jacquelyn-bradshaw — CHANGES_REQUESTED** [review]

File: client/package-lock.json
Comment (PAZHames) [reply client/package-lock.json:1]: yes this is what I was referring to. trying to figure that out

**Review by soojinhwang — APPROVED** [review]

**Review by oliviaportess — APPROVED** [review]

**Review by soojinhwang — APPROVED** [review]

**Review by soojinhwang — APPROVED** [review]

**Review by soojinhwang — APPROVED** [review]

**Review by jacquelyn-bradshaw — APPROVED** [review]

**Pull request merged**